### DISCUSSION TITLE
Cronjob creation from Marketplace

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#209] [Ideas]
Hi guys!

I would like to create a new cronjob starting from a Marketplace component.
The flow I follow to do this today is:
- create a new microservice in the microservice section, without saving. So, the Console creates my Git repo without adding it among the microservices of my project
- create a new cronjob starting from the docker image built in the previously created Git repo

I think the experience would be smoother, if we allow to have cronjob elements in the marketplace and let the user choose among these elements when he need to create a new cronjob.

What do you think?

Thanks!

**Comment by davidebianchi** [comment]
Yes, sure. We should enable the marketplace also for the CronJob component!

**Comment by fredmaggiowski** [comment]
I completely agree with you, we have it planned but have no use case for useful CronJobs to create from at the moment; do you have some interesting use case in mind?

**Reply by lucascanna** [reply]
Hi @fredmaggiowski, the use case that inspired this discussion is that we have to build multiple cronjobs that manipulates some S3 files. This is a very frequent use case when companies have lagacy systems that communicate with SFTP batches. They usually need cronjobs to move around the several input and output files.

**Comment by peppemanzi** [comment]
Duplicate of https://github.com/mia-platform/community/discussions/237
